[INVE-20296] No new enzyme test (#33)
sirensolutions/eslint-plugin: rules/no-new-enzyme-test/README.md

# Prevent adding a new enzyme test file

> This rule use git to get a list of all the new files in a commit.

Reviewer: Just being pedantic :D

```suggestion
This rule uses Git to get a list of all the new files being staged for commit.
```
